lib/Db.php

```diff
@@ -42,10 +42,11 @@ static function execute($query, $fetchStyle = PDO::FETCH_BOTH) {
   static function executeSqlFile($fileName) {
     $dsn = Config::get('general.database');
     $parts = Db::parseDsn($dsn);
-    $command = sprintf("cat {$fileName} | mysql -u %s -h %s %s",
+    $command = sprintf("cat {$fileName} | mysql -u %s -h %s %s --password=%s",
```

Comment:
Looks good for now. Some flavors of MySQL complain if you add the password on the command line and instead expect it in `.my.cnf` or `.mylogin.cnf`. We'll see if this becomes a problem.